### PR TITLE
Fluent fixes 6

### PR DESCRIPTION
**Pull request by kikostadinov**
Part of: https://github.com/telerik/kendo-themes-private/issues/26

Breadcrumb
- Breadcrumb item font-size is 14px. By design it is 18px. (changed in design to be 14px)

TreeList
- Margin between expand/collapse icon and item's text should be 8px
- Margin between sort icon and header text should be 8px
- Sort index icon is not aligned to sort order icon. In addition, the margin between the both icons should be 4px (margin by design is 2px)

### CODE REVIEW

**Review by JoomFX — APPROVED** [review]
Just a suggestion - we might want to double-check with the design team if the Breadcrumb font-size really needs to be `18px` or if it can be `16px` where we can use `--kendo-font-size-lg`.

File: packages/fluent/scss/treelist/_layout.scss [hunk 1]
```diff
@@ -44,6 +44,11 @@
         tr.k-footer + tr.k-footer td {
             border-top-width: 0;
         }
+
+        .k-i-collapse,
+        .k-i-expand {
```
Comment (Juveniel) [inline packages/fluent/scss/treelist/_layout.scss:49]: same as first comment

File: packages/fluent/scss/breadcrumb/_variables.scss [hunk 2]
```diff
@@ -21,7 +21,7 @@ $kendo-breadcrumb-border-width: 0px !default;
 $kendo-breadcrumb-font-family: var( --kendo-font-family, inherit ) !default;
 /// The font size of the breadcrumb
 /// @group breadcrumb
-$kendo-breadcrumb-font-size: var( --kendo-font-size, inherit ) !default;
+$kendo-breadcrumb-font-size: calc( var( --kendo-font-size, inherit ) + 4px) !default;
 /// The line height of the breadcrumb
 /// @group breadcrumb
 $kendo-breadcrumb-line-height: var( --kendo-line-height, normal ) !default;
```
[the remark this answers is not in the document]
Comment (kikostadinov) [reply packages/fluent/scss/breadcrumb/_variables.scss]: The line height is currently 20px, which corresponds with the design.

File: packages/fluent/scss/grid/_layout.scss [hunk 3]
```diff
@@ -435,6 +435,13 @@
             overflow: hidden;
             text-overflow: ellipsis;
             outline: 0;
+
+            .k-i-sort-desc-small,
+            .k-i-sort-asc-small,
+            .k-i-sort-desc-sm,
+            .k-i-sort-asc-sm {
```
[the remark this answers is not in the document]
Comment (kikostadinov) [reply packages/fluent/scss/grid/_layout.scss:442]: I had checked with her about this and she advised that we still need support for these classes.

File: packages/fluent/scss/treelist/_layout.scss (hunk 1 above)
Comment (kikostadinov) [reply packages/fluent/scss/treelist/_layout.scss:49]: The icons are with 16/16 size and I don't see a specified line height in the design.

File: packages/fluent/scss/grid/_layout.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (kikostadinov) [reply packages/fluent/scss/grid/_layout.scss]: After further investigation with @TeyaVes , looks like the specific sort classes will not be needed.

**Review by TeyaVes — APPROVED** [review]

File: packages/fluent/scss/treelist/_layout.scss (hunk 1 above)
Comment (Juveniel) [reply packages/fluent/scss/treelist/_layout.scss:49]: I was referring to my comment regarding using specific icon classes here

**Review by Juveniel — APPROVED** [review]

**Pull request merged**